### PR TITLE
Merging install pages for dd (125565)

### PR DESCRIPTION
**Pull request by jeff-colucci**

### CODE REVIEW

File: src/content/docs/tutorial-dd-migration/installing-agent.mdx [hunk 1]
```diff
@@ -0,0 +1,448 @@
...
+import journeyLogsDash2 from 'images/logs_screenshot-crop_journey-dash-two.webp'
+
+import journeyLogInstall from 'images/logs_screenshot-crop_journey-install.webp'
+
+The first step to trying out New Relic for yourself is ingesting some data to monitor. The most useful places to start are an infrastructure agent, application monitor (APM), or logging monitor. Once you do, you can try our dashboards and alerts for yourself, as well as a slew of other advanced features listed in the [final doc](/docs/tutorial-dd-migration/making-the-switch/#do-more) of this series. Nearly any feature you want to try will be available to you for free once an agent is installed: just choose one of our agents to install below to get started!
+
+<Tabs>
+	<TabsBar>
+        <TabsBarItem id="1">
+            Infrastructure Agent
```
Comment (akristen) [inline src/content/docs/tutorial-dd-migration/installing-agent.mdx:46]: are these supposed to be capitalized?

File: src/content/docs/tutorial-dd-migration/installing-agent.mdx [hunk 2]
```diff
@@ -0,0 +1,448 @@
...
+
+        That's where New Relic comes in. Not only do our guided installs and pre-configured dashboards and alerts help you get up and running quickly, our tools will help you proactively troubleshoot and improve your infrastructure. Whether you want to view one host or one thousand, you can do everything you need from one central location without having to hunt for the data you need.
+    </Side>
+</SideBySide>
+
+## Your infrastructure monitoring solution [#infra-solution]
+
+New Relic makes it easy for you to see what's going on inside your infrastructure, whether it runs in the cloud or dedicated hosts. In Datadog, you may be familiar with these capabilities as **agents**, and we call the same infrastructure capabilities **agents** at New Relic as well. Here's how we organize our infrastructure agents and integrations:
+
+* Infrastructure agents for Linux, macOS, and Windows operating systems with guided and manual install options.
```
Comment (akristen) [inline src/content/docs/tutorial-dd-migration/installing-agent.mdx:77]: these don't need terminal punctuation/periods

File: src/content/docs/tutorial-dd-migration/installing-agent.mdx [hunk 3]
```diff
@@ -0,0 +1,448 @@
...
+    title="Premade dashboards"
+    alt="An image displaying some of New Relic's premade infrastructure dashboards"
+    src={journeyNewDash}
+/>
+    </Side>
+    <Side>
+New Relic provides out-of-the-box dashboards from the moment you set up an infrastructure integration. These dashboards are pre-configured, allowing you to begin visualizing your data with zero setup time. 
+
+Some of the most important areas that our infrastructure dashboards automatically cover are:
+* Event data for your hosts, including metric data, the ability to filter by host, and notifications about your system health under the **Hosts** tab.
```
Comment (akristen) [inline src/content/docs/tutorial-dd-migration/installing-agent.mdx:380]: these don't need terminal punctuation

File: src/content/docs/tutorial-dd-migration/installing-agent.mdx [hunk 4]
```diff
@@ -0,0 +1,448 @@
...
+    title="Premade alerts"
+    alt="An image displaying some of New Relic's premade APM alerts"
+    src={journeyAPMDash2}
+/>
+    </Side>
+    <Side>
+Like with our infrastructure monitors, we give you access to out-of-the-box dashboards immediately upon setting up your APM integration. These dashboards are also pre-configured, giving you depth of information on a wide variety of data points. 
+
+Some of the most important areas that our APM dashboards automatically cover are:
+* Details on your core web vitals, across any time range you need on the **Summary** page, making app monitoring easier and more straightforward.
```
Comment (akristen) [inline src/content/docs/tutorial-dd-migration/installing-agent.mdx:404]: ibid re: terminal punct.

File: src/content/docs/tutorial-dd-migration/installing-agent.mdx [hunk 5]
```diff
@@ -0,0 +1,448 @@
...
+    title="Premade dashboards"
+    alt="An image displaying some of New Relic's premade infrastructure dashboards"
+    src={journeyLogsDash2}
+/>
+    </Side>
+    <Side>
+New Relic's log monitoring gives you a premade central location with a variety of information available from the moment you set up the integration. No setup time required: just install the monitor and begin finding the data you need quickly.
+
+The logs UI has four key sections:
+* An **All logs** summary page that displays all your ingested logs and gives you the option to sort, filter, and add them to dashboards for quicker future access.
```
Comment (akristen) [inline src/content/docs/tutorial-dd-migration/installing-agent.mdx:428]: ibid terminal punct

File: src/content/docs/tutorial-dd-migration/installing-agent.mdx [hunk 6]
```diff
@@ -0,0 +1,448 @@
...
+
+Once installed, your log monitoring is done in the **Logs** section of the UI, exactly like it is in Datadog. You can use our guided installations for logs after setting up any of our [infrastructure](/docs/journey-migration/migrating-infra/) or [APM](/docs/journey-migration/migrating-apm/) agents. 
+
+The steps for setting up logs varies depending on the data source, but no matter which you choose, you can rest assured knowing we'll guide you through every step of the process to make it as easy and straightforward as possible. You can see our list of guided installations for logs by selecting **one.newrelic.com > Add Data > Logging** in the New Relic UI.
+
+        </TabsPageItem>
+    </TabsPages>
+</Tabs>
+
+## Explore premade dashboards [#add-dashboard]
```
Comment (akristen) [inline src/content/docs/tutorial-dd-migration/installing-agent.mdx:359]: I'm not sure why but in the preview, this header looks weird and flush with the preceding paragraph

**Review by akristen — DISMISSED** [review]
Made some stylistic notes on the Install doc. I think it's a little text heavy, but I'm not sure what the solution is tbh. Something I'm wondering is if you should add 'Previous/Next' side by side tiles -- that makes looping from 'Making the switch' a little easier. The 'Next' tile can port the users to wherever you want them to go

File: src/content/docs/tutorial-dd-migration/installing-agent.mdx (hunk 1 above)
Comment (jeff-colucci) [reply src/content/docs/tutorial-dd-migration/installing-agent.mdx:46]: Fixed, thank you!

**Review by akristen — APPROVED** [review]
Looks good! Like the call out to link to support. Good call

**Pull request merged**